src/components/CommentForm.vue

```diff
@@ -0,0 +1,89 @@
+<script setup>
+import { ref, onMounted, watch } from 'vue'
+import { addComment } from '@/api/comments'
+import InputField from './InputField.vue'
+import TextAreaField from './TextAreaField.vue'
+
+const { postId, formName, formEmail } = defineProps({
+  postId: Number,
+  formName: String,
+  formEmail: String
+})
```

Comment:
You haven't added default values or validation. Adding validation helps make your component more resilient.